diff --git a/node_modules/react-scripts/config/webpack.config.js b/node_modules/react-scripts/config/webpack.config.js
index e465d8e..5de0b28 100644
--- a/node_modules/react-scripts/config/webpack.config.js
+++ b/node_modules/react-scripts/config/webpack.config.js
@@ -373,6 +373,19 @@ module.exports = function (webpackEnv) {
                 },
               },
             },
+            // https://github.com/eemeli/yaml/issues/208#issuecomment-720504241
+            {
+              test: /node_modules\/yaml\/browser\/dist\/.*/,
+              type: 'javascript/auto',
+              use: {
+                loader: 'babel-loader',
+                options: {
+                  presets: [
+                    '@babel/preset-env',
+                  ],
+                },
+              },
+            },
             // "url" loader works like "file" loader except that it embeds assets
             // smaller than specified limit in bytes as data URLs to avoid requests.
             // A missing `test` is equivalent to a match.
